vsintegration/src/vs/FsPkgs/FSharp.Project/Common.Source.CSharp/Project/FileNode.cs

```diff
@@ -1122,47 +1067,7 @@ private void RenameCaseOnlyChange(string newFileName)
 
         #endregion
 
-#region SingleFileGenerator Support methods
```

Comment:
I'm not sure about deleting the single-file-generator stuff, someone may want to take a stab at enabling this feature in the project system.  I'll let @latkin and @KevinRansom decide